policyengine_api/routes/household_routes.py

```diff
@@ -0,0 +1,199 @@
+from flask import Blueprint, Response, request
+import json
+
+from policyengine_api.constants import COUNTRY_PACKAGE_VERSIONS
+from policyengine_api.services.household_service import HouseholdService
+from policyengine_api.utils import hash_object
+from policyengine_api.utils.payload_validators import (
+    validate_household_payload,
+    validate_country,
+)
+
+
+household_bp = Blueprint("household", __name__)
+household_service = HouseholdService()
+
+
+@household_bp.route("/<household_id>", methods=["GET"])
+@validate_country
+def get_household(country_id: str, household_id: str) -> Response:
+    """
+    Get a household's input data with a given ID.
+
+    Args:
+        country_id (str): The country ID.
+        household_id (str): The household ID.
+    """
+    print(f"Got request for household {household_id} in country {country_id}")
+
+    # Ensure that household ID is a number
+    try:
+        household_id = int(household_id)
```

Comment:
Wow, that would make life easier. Let me see if I can find in logs, curious to see if there are more advanced features so that we could further cut down on validation code.

Reply:
Looks like Flask uses Werkzeug for routing, and this is documented in there at, e.g., https://werkzeug.palletsprojects.com/en/stable/routing/#werkzeug.routing.IntegerConverter